scripts/docker/DSS.Dockerfile

```diff
@@ -4,47 +4,59 @@ COPY dss-ansible/artifacts/dss-gcc510-*.rpm ./
 COPY dss-ansible/artifacts/aws-sdk-cpp-*.rpm ./
 RUN set -eux \
 	&& yum install -y \
-        epel-release centos-release-scl-rh && \
-    yum group install -y \
-        "Development Tools" && \
+        epel-release \
+        centos-release-scl-rh && \
     yum install -y \
```

Comment:
this can be removed along with preceding && as it is redundant

Reply:
Marking resolved per other comment